### PR TITLE
Target angular 16

### PR DESCRIPTION
**Pull request by PieterjanDeClippel**
## Summary

Target angular 16

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by PieterjanDeClippel** [comment]
Rebased just now

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jest-preset-angular",
-  "version": "13.0.1",
+  "version": "13.0.2",
```
Comment (michaelfaith) [inline package.json:3]: I believe @ahnpnl bumps the version at release time.  Not sure if it's appropriate to include it in this PR

**Comment by michaelfaith** [comment]
This project uses yarn 3, which uses a cache for dependencies.  So any changes to the `package.json` would need to also include updates to the `.yarn` cache and `yarn.lock`.  That's why the build is failing.

File: package.json (hunk 1 above)
Comment (ahnpnl) [reply package.json:3]: yes the version is bump during the release process

Comment (PieterjanDeClippel) [reply package.json]: Reset the version

**Comment by PieterjanDeClippel** [comment]
> This project uses yarn 3, which uses a cache for dependencies. So any changes to the `package.json` would need to also include updates to the `yarn.lock`. That's why the build is failing. You can run `yarn` to update the cache and lock file.

Ok

**Comment by henryruhs** [comment]
Excuse me, how does bumping the dependencies solve the NGCC issue - should be disabled for Angular 16.

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Jest: Got error running globalSetup - [...]/node_modules/jest-preset-angular/global-setup.js, reason: Package subpath './ngcc' is not defined by "exports" in [...]/node_modules/@angular/compiler-cli/package.json
```

**Comment by michaelfaith** [comment]
> Excuse me, how does bumping the dependencies solve the NGCC issue - should be disabled for Angular 16.
> 
> ```
> Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Jest: Got error running globalSetup - [...]/node_modules/jest-preset-angular/global-setup.js, reason: Package subpath './ngcc' is not defined by "exports" in [...]/node_modules/@angular/compiler-cli/package.json
> ```

It doesn't.  But there's a separate issue I opened for that, and you can just remove the `globa-setup` from your config to get past that, until more robust error handling can be implemented. https://github.com/thymikee/jest-preset-angular/issues/1987

**Comment by ahnpnl** [comment]
@PieterjanDeClippel  would you please run `yarn dedupe` and recommit the lock file?

**Comment by PieterjanDeClippel** [comment]
@ahnpnl Ran `yarn dedupe` - Rebased

**Review by michaelfaith — APPROVED** [review]
lgtm.  Thanks for taking this up

**Review by ahnpnl — APPROVED** [review]
Thanks!

**Comment by PieterjanDeClippel** [comment]
When will this be released? Waiting for a new NX version

**Pull request merged**